src/plugins/controls/common/options_list/suggestions_sorting.ts

```diff
@@ -10,13 +10,17 @@ import { Direction } from '@elastic/eui';
 
 export type OptionsListSortBy = '_count' | '_key';
 
-export const DEFAULT_SORT: SortingType = { by: '_count', direction: 'desc' };
+export const OPTIONS_LIST_DEFAULT_SORT: OptionsListSortingType = {
+  by: '_count',
+  direction: 'desc',
+};
```

Comment:
I noticed that, after [this comment ](https://github.com/elastic/kibana/pull/144867#discussion_r1031765527) was made for the sort PR, I only renamed the specific options list **types** - however, as I was working on this PR, I figured this rule should apply to  options list **constants** as well. Hence, renaming this to be more specific.